backend/config/LOG_LEVEL

```diff
@@ -1 +1 @@
-debug
\ No newline at end of file
+trace
```

Comment:
Was it intentional to commit this change?

Reply:
yes